### PR TITLE
[17.0][MIG] portal_account_personal_data_only: Migration to 17.0

### PR DESCRIPTION
**Pull request by SodexisTeam**

### CODE REVIEW

File: portal_account_personal_data_only/tests/test_partner_access.py [hunk 1]
```diff
@@ -36,10 +39,39 @@ def _create_partner(self, user):
...
+            .create(invoice_vals)
+        )
+        invoice.action_post()
+        return invoice
+
     def test_access_invoice(self):
-        invoice_a = self.create_invoice_partner(partner_id=self.partner_a.id)
-        invoice_b = self.create_invoice_partner(partner_id=self.partner_b.id)
-        invoice_c = self.create_invoice_partner(partner_id=self.partner_c.id)
+        invoice_a = self.create_invoice(partner_id=self.partner_a.id)
```
Comment (pedrobaeza) [inline portal_account_personal_data_only/tests/test_partner_access.py]: Isn't there already an invoice creation helper method in super class?

**Review by pedrobaeza — COMMENTED** [review]
/ocabot migration portal_account_personal_data_only

**Comment by SodexisTeam** [comment]
@pedrobaeza have made the mentioned changes, Please review it.

**Review by pedrobaeza — APPROVED** [review]

**Review by victoralmau — APPROVED** [review]
Code review OK

**Review by marcelsavegnago — APPROVED** [review]
LGTM

**Pull request merged**